src/ODC-By-1.0.xml

```diff
@@ -0,0 +1,504 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="ODC-By-1.0"
+            name="Open Data Commons Attribution License v1.0">
+      <crossRefs>
+         <crossRef>https://opendatacommons.org/licenses/by/1.0/</crossRef>
+      </crossRefs>
+    <text>
+      <titleText>
+	     <optional>##</optional>
+         <p>ODC Attribution License (ODC-By)</p>
+      </titleText>
+      <optional>###</optional>
+      <p>Preamble</p>
+      <p>The Open Data Commons Attribution License is a license agreement intended to allow users to freely
+      share, modify, and use this Database subject only to the attribution requirements set out in Section 4.</p>
+      <p>Databases can contain a wide variety of types of content (images, audiovisual material, and sounds
+      all in the same database, for example), and so this license only governs the rights over the Database,
+      and not the contents of the Database individually. Licensors may therefore wish to use this license
+      together with another license for the contents.</p>
+      <p>Sometimes the contents of a database, or the database itself, can be covered by other rights not
+      addressed here (such as private contracts, trademark over the name, or privacy rights / data protection
+      rights over information in the contents), and so you are advised that you may have to consult other
+      documents or clear other rights before doing activities not covered by this License.</p>
+      <p>------</p>
+    
+      <p>The Licensor (as defined below)</p>
+      <p>and</p>
+      <p>You (as defined below)</p>
+      <p>agree as follows:</p>
```

Comment:
I think this is all one paragraph.  I'd rather not force newlines, but if you want to, I think you should use [`<br/>`][1].

Also, should this text be optional?  It looks like it's "I'm about to start in on the license text", not like actual license text itself.

[1]: https://github.com/spdx/license-list-XML/blob/v3.1/schema/ListedLicense.xsd#L399-L407

Reply:
it's essentially how it appears in the original: https://opendatacommons.org/licenses/by/1.0/ and https://opendatacommons.org/files/2018/02/odc_by_1.0_public_text.txt

Reply:
> it's essentially how it appears in the original...

Using `<br/>` allows you to reproduce that appearance without claiming that this is four separate paragraphs.

Reply:
it doesn't matter, so let's not waste time on it - it's fine as is :)

Reply:
> it doesn't matter, so let's not waste time on it - it's fine as is :)

It matters to me, but I'm fine filing a follow-up PR ;).